LumpNav/LumpNav.py

```diff
@@ -261,11 +267,16 @@ def setupScene(self): #applet specific
       self.needleModelToNeedleTip.SetName("NeedleModelToNeedleTip")
       m = vtk.vtkMatrix4x4()
       m.SetElement( 0, 0, 0 )
-      m.SetElement( 1, 1, 0 )

```

Comment:
Yes, I forgot to mention this difference in my commit, sorry. It was a bit confusing when I started using LumpNav here on our system and the tools where not aligned properly in the 3D view.

If I would like it to be a part of the .ini file, where do you suggest to give the user the option of changing the matrix? Since I assume I need to then include the use of writeTransformToSettings and readTransformFromSettings. For example, would it be a good idea to have the values you are using at Queen's as default (in the .ini file) and then a rotate button that saves the rotated transform to the .ini file?